hopsworks-admin/src/main/java/io/hops/hopsworks/admin/project/ProjectsManagementBean.java

```diff
@@ -426,4 +441,20 @@ public String getCondaCommands(String projectname) {
     }
     return sb.toString();
   }
+  
+  public void deleteProject() {
+    LOG.log(Level.INFO, "Deleting project: " + toBeDeletedProject);
+    String sessionId = ((HttpServletRequest) FacesContext.getCurrentInstance().getExternalContext().getRequest())
+        .getSession().getId();
+    try {
+      projectsManagementController.deleteProject(toBeDeletedProject, sessionId);
```

Comment:
@kouzant This sessionId is the Admin's? It's used here later on https://github.com/kouzant/hopsworks/blob/9673b75257bc87ecea6a1609c83ae6faae445d26/hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java#L989 which should be the session of the project's owner? Maybe I'm missing something here.

Reply:
That's true. That's the session ID of the admin user. We use the session further in the project deletion to check if the user associated with this session ID a) has authenticated with Hopsworks b) has a valid role in the project. Case a is covered since the Admin user is logged-in. Case B I have to change it in https://github.com/hopshadoop/hopsworks/blob/master/hopsworks-api/src/main/java/io/hops/hopsworks/api/zeppelin/rest/InterpreterService.java#L98 and add a check if the user is HOPS_ADMIN

Reply:
Also here https://github.com/hopshadoop/hopsworks/blob/master/hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java#L993 we make a REST call to restart the interpreter but I can't find the endpoint. @ErmiasG what am I missing?

Reply:
@kouzant https://github.com/hopshadoop/hopsworks/blob/master/hopsworks-api/src/main/java/io/hops/hopsworks/api/zeppelin/rest/InterpreterRestApi.java#L526